x/concentrated-liquidity/internal/math/math.go

```diff
@@ -81,20 +95,21 @@ func CalcAmount1Delta(liq, sqrtPriceA, sqrtPriceB sdk.Dec, roundUp bool) sdk.Dec
 // else
 // sqrtPriceNext = ((liquidity)) / (((liquidity) / (sqrtPriceCurrent)) + (amountRemaining))
 func GetNextSqrtPriceFromAmount0RoundingUp(sqrtPriceCurrent, liquidity, amountRemaining sdk.Dec) (sqrtPriceNext sdk.Dec) {
-	numerator := liquidity
-	product := amountRemaining.Mul(sqrtPriceCurrent)
+	if amountRemaining.Equal(sdk.ZeroDec()) {
```

Comment:
Can we remove pseudocode in the spec please and, instead, explain what that means?

Reply:
Having the formulas in the godoc has been really helpful for me to reference. I have referenced quite a few times.

Reply:
It doesn't seem like a formula, more like a code block/pseudocode which shouldn't be in the spec.

The spec should abstract away the implementation details, allowing the developer to treat the function as a black box. In the current state, it only feels like additional complexity trying to visually QA the spec first before reading the function implementation. 

I agree that formulas are useful. However, if we are to leave them in the codebase, it should be clear what they are referring to exactly. I'm supportive of having them if they are clearly referencing the code lines and we don't have to include large chunks of pseudocode in the spec

Reply:
Removed